tests/virt_autotest/libvirt_extend_storage_lvm.pm

```diff
@@ -33,11 +32,7 @@ our $lvm_vg_name = 'lvm_vg';
 our $lvm_pool_name = 'guest_image_lvm';
 sub run_test {
     my ($self) = @_;
-
-    record_info "Prepare Guest Systems";
-    foreach (keys %virt_autotest::common::guests) {
-        start_guests() unless is_guest_online($_);
-    }
```

Comment:
Please help make sure removing this block does not lead to unexpected test run failure.